src/sender.rs

```diff
@@ -526,7 +531,7 @@ where
                 .await?;
 
             let messages = OutgoingPushMessages {
-                destination: recipient.uuid,
+                destination: recipient.raw_uuid(),
```

Comment:
Was this really raw-uuid-string only even before? Then again, I don't think we've ever supported ACI-to-PNI sending...

Reply:
It might just work ™️ if we make the types fit correctly.